package.json

```diff
@@ -11,11 +11,13 @@
     "build-prod-readable": "yarn run build:clean && NODE_ENV=production webpack --progress",
     "build-docs": "documentation build src/types/profile.js -f md > docs/processed-profile-format.md; cat docs/processed-profile-format.md",
     "eslint": "eslint index.js src",
```

Comment:
Debugger is using `yarn run lint` instead of `yarn run eslint`. Maybe this would be a good thing for us to adopt as well? That way it's using the action we're doing and not the vendor's package name.